src/Sulu/Bundle/SecurityBundle/Resources/js/containers/RoleAssignments/RoleAssignment.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import React from 'react';
+import {observer} from 'mobx-react';
+import {MultiSelect} from 'sulu-admin-bundle/components';
+import type {Localization} from 'sulu-admin-bundle/stores';
+import roleAssignmentsStyle from './roleAssignments.scss';
```

Comment:
Any reason to not split that into `roleAssignment.scss` and `roleAssignments.scss`? In case these files grow they would be already split then.